### PR TITLE
Remove revisions_index view

### PR DESCRIPTION
**Pull request by gasman**
The revisions_index view was superseded by the page history view in Wagtail 2.10, but it had been left in place and still linked from one somewhat obscure place: the "No, don't unschedule" button on the "confirm unschedule revision" screen. :-) This PR removes it properly, replacing it with a redirect to the page history view.

### CODE REVIEW

File: wagtail/admin/views/pages/revisions.py [hunk 1]
```diff
@@ -16,26 +15,8 @@
...
-    ordering = request.GET.get('ordering', '-created_at')
-    if ordering not in ['created_at', '-created_at', ]:
-        ordering = '-created_at'
-
-    revisions = page.revisions.order_by(ordering)
-
-    paginator = Paginator(revisions, per_page=20)
-    revisions = paginator.get_page(request.GET.get('p'))
-
-    return TemplateResponse(request, 'wagtailadmin/pages/revisions/index.html', {
```
Comment (zerolab) [inline wagtail/admin/views/pages/revisions.py:33]: The revisions index template is still there.

**Review by zerolab — CHANGES_REQUESTED** [review]
We left the revisions index as a fallback, but so happy you remembered about it and cleaned it up.

One remaining template to purge and this is good to go!

File: wagtail/admin/views/pages/revisions.py (hunk 1 above)
Comment (gasman) [reply wagtail/admin/views/pages/revisions.py:33]: Good catch! Updated to remove that one now.

**Review by zerolab — APPROVED** [review]
LGTM. Thank you @gasman

**Pull request merged**